app/main/views/services.py

```diff
@@ -452,17 +457,29 @@ def update_section_submission(framework_slug, lot_slug, service_id, section_id,
             "services/edit_submission_section.html",
             framework=framework,
             section=section,
+            next_section_name=get_next_section_name(content, section.id),
             service_data=update_data,
             service_id=service_id,
             one_service_limit=has_one_service_limit(lot_slug, framework['lots']),
+            return_to_summary=bool(request.args.get('return_to_summary')),
             errors=errors
         )
 
-    return redirect(url_for(".view_service_submission",
-                            framework_slug=framework['slug'],
-                            lot_slug=draft['lot'],
-                            service_id=service_id,
-                            _anchor=section_id))
+    return_to_summary = bool(request.args.get('return_to_summary'))
+    next_section = content.get_next_editable_section_id(section_id)
+
+    if next_section and not return_to_summary and request.form.get('continue_to_next_section'):
+        return redirect(url_for(".edit_service_submission",
```

Comment:
We need to get a call on this from Cath - restoring this flow is what's in the story. 

From what I understand users *do* want to be able to get through the questions - many of them will collate the answers to all the questions in a spreadsheet, talking to whoever is needed across the organisation to get the right answers, and then someone will sit and "data entry" them into the Marketplace.  If that's a flow that many suppliers use then we should support it.